jisantuc/AFDG
# Render oafs and wizards from tiles

Overview
-----

This PR adds game units, the ability to render them, and buttons to enter a mode
where you can add them to tiles.

Testing
-----

- bring up the application
- click `add oafs` or `add wizards`
- click on some tiles and look check it out you have some units

Closes #15
